### ISSUE TITLE
mariadb. Could not start, cannot install

### ISSUE THREAD

**Issue by mariusebastian** [home-assistant/hassio-addons#1024]
I'm running hassio on has HassOS 3.7, homeassistant version 0.104.3 on a RPi3b

I had hassio running with the core-mariadb addon for about a month. Last night i noticed hassio was offline, I found no way of reconnecting without a hard reboot. After this the recorder, history and logbook failed to start, evidently because the mariadb addon was stopped. When I tried to start it via hassio it was unresponsive, no info in the app log. I tried changing both the yaml and the app config, but the addon still refused to start. Tried uninstalling the addon, and now I can't re-install it even after a reboot. The install button just flashes red a few seconds after I try to install.

**Comment by frenck** [comment]
Any information in the Supervisor logs? (Hass.io panel -> System tab -> Logs)

**Comment by mariusebastian** [comment]
Hm, the below entries seem relevant

`20-01-27 10:58:07 INFO (SyncWorker_14) [hassio.docker.interface] Pull image homeassistant/armv7-addon-mariadb tag 1.3.

20-01-27 10:58:09 ERROR (SyncWorker_14) [hassio.docker.interface] Can't install homeassistant/armv7-addon-mariadb:1.3 -> 500 Server Error: Internal Server Error ("layer does not exist").
`

**Comment by frenck** [comment]
That is odd, something wrong with Docker probably.

We have a command for that: `hassio supervisor repair`

Please be aware that this command can take a moment to complete. Furthermore, if you have installed Docker containers manually outside of Hass.io, that command will remove those manually added containers. If you haven't done that and just use Hass.io + add-ons, the repair command might help you.

**Comment by mariusebastian** [comment]
I ran the command, but still no luck.
I also noticed that I have lost remote access via my duckdns domain. The duckdns addon was running without a glitch before this crash event, but now it seems to have a problem. The duckdns addon log does not show any problems, the cert is checked, renew is skipped because I have more than 30 days left, and after that it's just INFO: OK NOCHANGE events. No errors in the supervisor log either, other than the new failed attempt to install mariadb...
